### PR TITLE
Config based reset filter

### PR DESCRIPTION
**Pull request by phndiaye**
### What does this PR do?

Do not show the Reset Filters button if the filtering feature isn't on.

### 🧑‍💻 Developer Heads Up

⚡ Since we are using [Ember Octane](https://blog.emberjs.com/octane-is-here/) now:
* Feel free to migrate existing components to Glimmer Components.
* Write new ones exclusively in it.

Useful Resource : [Ember Octane vs Classic Cheat Sheet](https://ember-learn.github.io/ember-octane-vs-classic-cheat-sheet/)

### Good PR checklist

- [x] Title makes sense
- [x] Is against the correct branch
- [x] Only addresses one issue
- [x] Properly assigned
- [ ] Added/updated tests
- [ ] Added/updated documentation
- [ ] Migrated touched components to Glimmer Components
- [x] Properly labeled

### CODE REVIEW

**Review by Hegazy360 — APPROVED** [review]

File: index.js [hunk 1]
```diff
@@ -1,12 +1,37 @@
...
+    if (vendorTree) {
+      trees.push(vendorTree);
+    }
+
+    trees.push(
+      writeFile(
+        '@upfluence/hypertable/register-version.js',
+        `Ember.libraries.register('${name}', '${version}');`
+      )
+    );
```
Comment (phndiaye) [inline index.js:33]: This is to display the Hypertable version in the Ember inspector.

File: app/templates/components/hyper-table.hbs [hunk 2]
```diff
@@ -27,12 +27,14 @@
...
-      data-control-name="table_reset_filters"
-      {{action "resetAllFilters"}}>
-      Reset Filters
-    </button>
+    {{#if manager.options.features.filtering}}
+      <button
+        class="margin-right-xx-sm margin-bottom-xxx-sm upf-btn upf-btn--default upf-btn--small"
+        data-control-name="table_reset_filters"
+        {{action "resetAllFilters"}}>
+        Reset Filters
```
Comment (JulienVannier66) [inline app/templates/components/hyper-table.hbs:35]: You know what I am going to say: can you add to transalation ? :)

Comment (phndiaye) [reply app/templates/components/hyper-table.hbs:35]: leave me alone, sir !

**Review by JulienVannier66 — APPROVED** [review]
Otherwise, LGTM

**Pull request merged**